mods/beds/functions.lua

```diff
@@ -75,15 +77,16 @@ local function lay_down(player, pos, bed_pos, state, skip)
 		player:set_eye_offset({x = 0, y = 0, z = 0}, {x = 0, y = 0, z = 0})
 		player:set_look_horizontal(math.random(1, 180) / 100)
 		default.player_attached[name] = false
-		player:set_physics_override(1, 1, 1)
+		player:set_physics_override(beds.player[name])
 		hud_flags.wielditem = true
 		default.player_set_animation(player, "stand" , 30)
+		beds.player[name] = nil
 
 	-- lay down
 	else
 		beds.pos[name] = pos
 		beds.bed_position[name] = bed_pos
-		beds.player[name] = 1
+		beds.player[name] = player:get_physics_override()
```

Comment:
Also, is `beds.player` part of the official API? Could anyone be relying on it?

Reply:
It still has the same falsy/truthy value, so it should be fine.